apps/pie-microsite/.eleventy.js

```diff
@@ -21,6 +21,9 @@ module.exports = eleventyConfig => {
   // Shortcodes
   shortcodes.addAllShortCodes(eleventyConfig);
 
+  // Pass CNAME file into dist output (needed for domain to work correctly)
+  eleventyConfig.addPassthroughCopy("CNAME");
```

Comment:
Super minor but single quotes here. Also it might be good to start thinking about grouping the passthroughs / config changes soon so they are separated out into logical directories?

Reply:
Sorry I only just noticed this and I completely agree - next change I can group them. Didn't actually spot the pre-existing one!